src/styles/blocks/_products-main.scss

```diff
@@ -0,0 +1,23 @@
+.main {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  grid-template-rows: repeat(2, 408px);
+  gap: 46px 48px;
+
+  height: 862px;
```

Comment:
```suggestion

```

Reply:
Not resolved. The height of the main content should be inherited from the content